openshift/cloud-deploy/secrets-template-aws.yaml

```diff
@@ -9,57 +9,57 @@ kind: Template
+#    host: bayesian_db_host
+#    port: bayesian_db_port
+#- apiVersion: v1
+#  kind: Secret
+#  metadata:
+#    name: anitya-postgres
+#  type: Opaque
+#  data:
+#    username: bayesian_db_username
+#    password: bayesian_db_password
+#    database: YW5pdHlh  # anitya
+#    initial-database: bayesian_db_database
+#    host: bayesian_db_host
+#    port: bayesian_db_port
 - apiVersion: v1
   kind: Secret
   metadata:
     name: aws
   type: Opaque
   data:
-    sqs-access-key-id: bayesian_sqs_aws_access_key_id
-    sqs-secret-access-key: bayesian_sqs_aws_access_key
-    s3-access-key-id: bayesian_s3_aws_access_key_id
-    s3-secret-access-key: bayesian_s3_aws_access_key
-    dynamodb-access-key-id: bayesian_dynamodb_aws_access_key_id
-    dynamodb-secret-access-key: bayesian_dynamodb_aws_access_key
-    emr-access-key-id: bayesian_emr_aws_access_key_id
-    emr-secret-access-key: bayesian_emr_aws_access_key
+#    sqs-access-key-id: bayesian_sqs_aws_access_key_id
+#    sqs-secret-access-key: bayesian_sqs_aws_access_key
+#    s3-access-key-id: bayesian_s3_aws_access_key_id
+#    s3-secret-access-key: bayesian_s3_aws_access_key
+    dynamodb-access-key-id: QUtJQUpHRVhZS1ZFSTJZS09KT1E= 
+    dynamodb-secret-access-key: Z09FN3NsSHRvWEtPTlMrd1dMQzJCT0RHZndSZzJxQjFyMEtsTFM1dg==
+#    emr-access-key-id: bayesian_emr_aws_access_key_id
+#    emr-secret-access-key: bayesian_emr_aws_access_key
     sync-s3: MQ== # 1
-- apiVersion: v1
-  kind: Secret
-  metadata:
-    name: user-auth
-  type: Opaque
-  data:
-    jwt-pubkey: |
-      LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FR
-      OEFNSUlCQ2dLQ0FRRUFzNzNJQlRvMHJYMm05YkdKR09GcQpOdEQyWGlONlkzbUxqWURuQklMUkhW
-      UTNmeVpuVHkvcFJDK2FLUWwvckZiSmJ2MGNYSC9XZHFPVUd2NG81Y3NWCmNhUjdDUFdQUE5KZzRS
-      cmtndHJKR0FZNVp4dTBBNFNmbHlVSTZSTW54YnJsZVAvMytiSGpTNVc0eEZVdFgvdUwKOFVtK3ds
-      d3VSMjZ0UGVRQWU1SHlHTm8vbW1ETnpxb2hRR1ZjYTg5cUtmL0hGbkttWUxlTWNhV1FBSC9vMEtT
-      Swp5WlZFS2xHNjg5eTNLMFRxNlhZQkMrU2tPSXJzV2NnNzFaclg2YXptOERQQmE2L2hTY2s2MTlI
-      K0lMZTdWd2pYCnBaNHNTNXNMbzEwRTBzU0hOWmI1N284TXBHVFBCSFFQZ0tOTm5IR2xUbjJNeXpt
-      TFBabThPQ3IrS3VGdG1DeE8KTXdJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0t
+#- apiVersion: v1
+#  kind: Secret
+#  metadata:
+#    name: user-auth
+#  type: Opaque
+#  data:
+#    jwt-pubkey: |
+#      LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FR
+#      OEFNSUlCQ2dLQ0FRRUFzNzNJQlRvMHJYMm05YkdKR09GcQpOdEQyWGlONlkzbUxqWURuQklMUkhW
+#      UTNmeVpuVHkvcFJDK2FLUWwvckZiSmJ2MGNYSC9XZHFPVUd2NG81Y3NWCmNhUjdDUFdQUE5KZzRS
+#      cmtndHJKR0FZNVp4dTBBNFNmbHlVSTZSTW54YnJsZVAvMytiSGpTNVc0eEZVdFgvdUwKOFVtK3ds
+#      d3VSMjZ0UGVRQWU1SHlHTm8vbW1ETnpxb2hRR1ZjYTg5cUtmL0hGbkttWUxlTWNhV1FBSC9vMEtT
+#      Swp5WlZFS2xHNjg5eTNLMFRxNlhZQkMrU2tPSXJzV2NnNzFaclg2YXptOERQQmE2L2hTY2s2MTlI
+#      K0lMZTdWd2pYCnBaNHNTNXNMbzEwRTBzU0hOWmI1N284TXBHVFBCSFFQZ0tOTm5IR2xUbjJNeXpt
+#      TFBabThPQ3IrS3VGdG1DeE8KTXdJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0t
```

Comment:
I don't understand whole this `secrets-template-aws.yaml` change. Can you shed some light on it please ?